src/Phinx/Console/Command/Init.php

```diff
@@ -77,8 +77,29 @@ protected function execute(InputInterface $input, OutputInterface $output)
             ));
         }
 
+        // set the config format
+        $getFormat = new ChoiceQuestion(
+            'Would you like Phinx to use json yml or php based configuration?',
+            array('php', 'json', 'yml'),
+            2
+        );
+
+        $type = $helper->ask($input, $output, $getFormat);
+
+        switch ($type) {
+            case 'php':
+            case 'json':
+            case 'yml':
+                break;
+            default:
+                throw new \InvalidArgumentException(sprintf(
+                    'The format "%s" is not one of json, php or yml',
```

Comment:
Yeah your right, I wasn't using ChoiceQuestion at first but forgot to get rid of the check.